metrics/base-metrics/src/test/java/org/eclipse/microprofile12/metrics/base/BaseMetricsTest.java

```diff
@@ -55,10 +55,10 @@ public void testBasePrometheusMetrics() throws IOException {
                 .request(TEXT_PLAIN)
                 .get();
         String responseText = response.readEntity(String.class);
-        assertTrue(responseText.contains("# TYPE base:thread_max_count"));
-        assertTrue(responseText.contains("# TYPE base:thread_count"));
-        assertTrue(responseText.contains("# TYPE base:cpu_system_load_average"));
-        assertTrue(responseText.contains("# TYPE base:jvm_uptime"));
+        assertTrue(responseText.contains("# TYPE base:thread_max_count") || responseText.contains("# TYPE base_thread_max_count gauge"));
```

Comment:
Remove ` gauge` from the end of the 2nd alternative.

Reply:
I mean, it obviously _is_ a gauge, so it technically isn't wrong. I'm asking to remove it just for consistency with the other assertions.